File: HELLO_PYTHON/day06/mysql_insert.py
import pymysql
from sqlalchemy.dialects.mysql.pymysql import MySQLDialect_pymysql
 
# MySQL Connection 연결
conn = pymysql.connect(host='localhost', user='root', password='python',
                       db='python', port=3305, charset='utf8')
 
curs = conn.cursor()
 
sql = """insert into emp
        (e_id, e_name, gen, addr)
        values (%s,%s,%s,%s)
"""

curs.execute(sql, ('4','4','4','4'))
conn.commit()
   
curs.close()
conn.close()                

File: HELLO_NODE2/mysql_insert.js
const mysql      = require('mysql'); //import 역할
const connection = mysql.createConnection({
  host     : 'localhost',
  user     : 'root',
  password : 'python',
  database : 'python',
  port : 3305 //기본 포트가 3306이므로 포트 지정도 해주어야 한다.
});


connection.connect(function(err) {
  var sql = "INSERT INTO emp VALUES ('4', '4', '4', '4')";
  connection.query(sql, function (err, result) {
    console.log("1 record inserted");
    console.log(result.affectedRows);
	
  });
});

	

